### DISCUSSION TITLE
No group output available? --cfg-group-output: 1?

### DISCUSSION THREAD

**Discussion by mnoesel** [uploadcare/file-uploader#336] [blocks] [answered]
Hi,
in the docs I read that group-output ist just available with the setting "0". Does this mean that it is never available for the blocks? Or not yet? When will be group-output enabled?

**Comment by nd0ut** [comment, marked as answer]
Hey @mnoesel,

you could enable group output with css:

```css
.config {
  --ctx-name: 'my-uploader';
  --cfg-group-output: 1;
}
```

and then consume it from the `<lr-data-output />` block:

```html
<lr-data-output id="output" class="config" use-event></lr-data-output>
```

```js
document.querySelector('#output').addEventListener('lr-data-output', (e) => {
  console.log(e.detail.data.groupData.uuid)
})
```

Note that for now, the group info won't be updated after using the image editor.
